src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Random RANDOM = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```
Don’t make instance constant field ;)